src/components/detail/Detail.jsx

```diff
@@ -56,8 +56,8 @@ class Detail extends React.Component {
   render() {
     const { state, props } = this;
     const overview = window.location.pathname.split('/')[1];
-    const canDeprecate = this.props.inputItems.find(item => item.id === identifiers.DEPRECATED);
-    const isDeprecated = canDeprecate && canDeprecate.value;
+    const active = this.props.inputItems.find(item => item.id === identifiers.ACTIVE);
+    const canDeprecate = active && active.value && props.deprecate;
```

Comment:
I think you can keep the isDeprecate, since it is deprecated or isn't. We are not doing quantum computing so there is no could be 😄